public/index.php

```diff
@@ -28,10 +28,10 @@
 			<img src="#" alt="">
 		</div>
 		<ul class="nav-area">
-			<li><a href="#">Home</li>
-			<li><a href="#">About</li>
-			<li><a href="#">Login</li>
-			<li><a href="#">Signup</li>
+			<li><a href="#">Home</a>
```

Comment:
forgot li tags

Reply:
Added the LI tags to the end, thank you for catching that!